repair_tests/preview_repair_test.py

```diff
@@ -18,6 +20,21 @@ def assert_no_repair_history(self, session):
         rows = session.execute("select * from system_distributed.parent_repair_history")
         assert rows.current_rows == []
 
+    @since('4.0')
+    def test_parent_repair_session_cleanup(self):
+        """
+        Calls incremental repair preview on 3 node cluster and verifies if all ParentRepairSession objects are cleaned
+        """
+        self.cluster.populate(3).start()
+        session = self.patient_cql_connection(self.cluster.nodelist()[0])
+        create_ks(session, 'ks', 2)
```

Comment:
why the replication should be 2 and not 1?

Reply:
Bc otherwise we'd noop [here](https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/service/StorageService.java#L3981).